src/app.rs

```diff
@@ -127,7 +127,11 @@ impl App {
         let root_hash = identity_manager.latest_root().await?;
         let root_hash = root_hash.into();
 
-        database.mark_root_submitted_to_contract(&root_hash).await?;
+        // We don't store the initial root in the database, so we have to skip this step
```

Comment:
Not really something we have to do right now, but it would be good to include a healthcheck for "does every identity have a root" somewhere on startup. Maybe even a separate monitor thread that checks that periodically and sounds an alarm if this happens? Even trying to insert these roots retroactively. This problem can easily arise if we have DB connection issues and this is an important stability assumption.